galleries/examples/color/individual_colors_from_cmap.py

```diff
@@ -0,0 +1,61 @@
+"""
+===========================================
+Selecting individual colors from a colormap
+===========================================
+
+Sometimes we want to use more colors or a different set of colors than the default color
+cycle provides. Selecting individual colors from one of the provided colormaps can be a
+convenient way to do this.
+
+Once we have hold of a `.Colormap` instance, the individual colors can be accessed
+simply using its ``.colors`` attribute.  If we want a specific number of colors taken
+at regular intervals from a continuous colormap, we can create a new colormap using the
+`~.Colormap.resampled` method.
+"""
+
+import matplotlib.pyplot as plt
+
+import matplotlib as mpl
+
+n_lines = 21
+
+cmap = mpl.colormaps.get_cmap('plasma').resampled(n_lines)
+colors = cmap.colors
```

Comment:
I note that the `colors` attribute does not appear in the API docs.  So perhaps I am not supposed to use it and should do `cmap(i)` instead?  I find the attribute quite intuitive though.

Reply:
This seems a bit confusing to me. Colors won't work at all on linear segmented colormaps, right?

Reply:
Ah OK, better go back to `cmap(i)` then.